internal/drivers/nomad/driver.go

```diff
@@ -196,7 +196,7 @@ func (p *config) Create(ctx context.Context, opts *types.InstanceCreateOpts) (*t
 	err = p.checkTaskGroupStatus(initJobID, initTaskGroup)
 	if err != nil {
 		defer p.Destroy(context.Background(), []*types.Instance{instance}) //nolint:errcheck
-		return nil, fmt.Errorf("scheduler: init job failed with error: %s", err)
+		return nil, fmt.Errorf("scheduler: init job failed with error: %s on Ip: %s", err, ip)
```

Comment:
s/Ip/IP

Is it okay to print IPs in the logs? We removed them from init log I think.

Reply:
IP is already logged in logs. Currently it is hard to debug without ip details